### DISCUSSION TITLE
Taskbar small icons

### DISCUSSION THREAD

**Discussion by oldprog** [valinet/ExplorerPatcher#757] [Ideas]
There is in Win 11 registry a switch in the explorer section to enable taskbar small icons.  Set value to 1.  I couldn't find this switch in settings where it was previously so you might want to add it to your menu.   Tested ok.  BTW - great product!

**Comment by valinet** [comment]
What do you mean exactly? What taskbar: 10 or 11? 

This is for the 10 taskbar: https://github.com/valinet/ExplorerPatcher/wiki/All-features#taskbar-icon-size.

For the 11 taskbar, it's not included on purpose, as it sometimes breaks things, like clock alignment etc; the option is not finished by Microsoft and not really supported.

**Comment by oldprog** [comment]
This was for the 11 taskbar.   It was available for the 10 and I used it.  It does hide the clock, but looks neater with toolbars, so I was looking for that again.

**Reply by valinet** [reply]
I don’t understand what you want exactly, please check the provided link in the previous message and see if it’s of any help. Thanks.

**Reply by Xroly** [reply]
Hi, I think I'm having the same issue with the new taskbar.
In Windows 10 when you launch any app, the icon is the size of the logo.
In Windows 11 with your cool taskbar you can't resize the icons, they are longer with text in it.
Is there a way to get back to small icons in the taskbar without changing the size of the taskbar itself ?
I added 2 screenshots below (maybe it will help you to understand)
![image](https://user-images.githubusercontent.com/100819283/156489491-cab26472-c394-47d5-aead-33e455924784.png)
![image](https://user-images.githubusercontent.com/100819283/156489736-aceacdaa-dac6-4ae0-b663-f9f117d6976b.png)

**Reply by valinet** [reply]
It helps reading the manual: https://github.com/valinet/ExplorerPatcher/wiki/All-features. I haven't written it in vain, or so I hoped. The option is in Windows since forever and among the first ones in the Properties UI:

![image](https://user-images.githubusercontent.com/6503598/156496162-174c1c54-6e14-4897-aee8-783ad636458f.png)

This is called always/never combine. Taskbar small icons is something entirely different. This:

![image](https://user-images.githubusercontent.com/6503598/156496235-2dca29bc-c4c9-4dc9-b988-394ccc8e3f77.png)

Versus this:

![image](https://user-images.githubusercontent.com/6503598/156496261-ba79e200-fc9b-46b6-9a13-3ef31ec6ea65.png)

**Reply by Xroly** [reply]
I read the manual and didn't any options about what I need.
I want to keep the icons in "never combine mode" but remove the text in it, the only way I found to remove the text is the "combine mode".
So is there a way to keep "never combine mode" without any text in each app ?
![image](https://user-images.githubusercontent.com/100819283/156616472-dd2dc10a-3219-4170-8d75-717c932fd2e4.png)

**Reply by Poopooracoocoo** [reply]
@xroly does this help? https://github.com/valinet/ExplorerPatcher/issues/767#issuecomment-1027926850

**Reply by Xroly** [reply]
Yesss, that's exactly want I wanted, thank you so much :)
Will there be this feature included directy in the ExplorerPatcher someday ? (without 7+ taskbar tweaker)

**Reply by Poopooracoocoo** [reply]
@Xroly why tho? Just use 7+ TT. The author of 7+ TT recommends using EP to get the W10 taskbar on W11. There's already a solution and there's no point in putting more stress on the dev IMO

**Reply by valinet** [reply]
@Xroly Exactly as @Poopooracoocoo said.
